dockerfile_functions.sh

```diff
@@ -698,33 +698,69 @@ print_cmd() {
 	fi
 }
 
-copy_scc_script() {
+print_scc_gen() {
 	if [[ "${vm}" == "openj9" && "${os_family}" != "windows" ]]; then
-	cat >> "$1" <<-EOI
-
-COPY generate_openj9_scc.sh /usr/local/bin/
-
-EOI
-	fi
-}
-
-run_scc_gen() {
-	if [[ "${vm}" == "openj9" && "${os_family}" != "windows" ]]; then
-		# Alpine needs bash to be present to run the generate_openj9_scc.sh script
 		if [[ "${os_family}" == "alpine" ]]; then
 			cat >> "$1" <<-EOI
-RUN apk add --no-cache --virtual .scc-deps bash curl; \\
-    /usr/local/bin/generate_openj9_scc.sh; \\
-    apk del --purge .scc-deps; \\
-    rm -rf /var/cache/apk/*;
+RUN apk add --no-cache --virtual .scc-deps bash curl
+EOI
+		fi
+		cat >> "$1" <<-EOI
+SHELL ["/bin/bash", "-c"]
+
+RUN set -euo pipefail; \\
+	unset OPENJ9_JAVA_OPTIONS; \\
```

Comment:
Looking closely, it appears that this entire RUN statement is using tabs. We have to use spaces for all the Dockerfile commands and tabs for the shell script. Please change it accordingly.